internal/logs/fetcher.go

```diff
@@ -67,5 +70,11 @@ func (f *FetcherForTestingPods) Logs(result oct.TestResult) (string, error) {
 		}
 	}
 
-	return logs.String(), nil
+	return strip(logs.String()), nil
+}
+
+// strip removes ANSI escape sequences from the input string
+func strip(s string) string {
```

Comment:
Why not call the function `stripANSI` and drop the comment?

Also a unit test for the function would be great.

Reply:
Yep I have updated the code based on your review